src/screens/FeedScreen/Content.js

```diff
@@ -255,6 +255,13 @@ const Content = ({
     });
   };
 
+  function getTopicsPositionBasedOnPostType() {
+    if (item.post_type === POST_TYPE_POLL && item.pollOptions.length > 3) {
+      return 'relative';
+    }
+    return 'absolute';
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getTopicsPositionBasedOnPostType` checks if the post type is a poll and if it has more than 3 options, then returns 'relative', otherwise 'absolute'. This logic seems to be correct based on the provided context. However, it would be better to handle the case where `item.pollOptions` is undefined or null to prevent potential errors.

```diff
  function getTopicsPositionBasedOnPostType() {
    if (item.post_type === POST_TYPE_POLL && item.pollOptions?.length > 3) {
      return 'relative';
    }
    return 'absolute';
  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->